device/globals/game.gd

```diff
@@ -257,6 +264,15 @@ func scene_input(event):
 
 
 	if event.is_action("menu_request") && event.is_pressed() && !event.is_echo():
+		# Do not display overlay menu with action menu or inventory, it looks silly and weird
+		if action_menu:
+			if action_menu.is_visible():
+				action_menu.stop()
+
+			# Hide inventory only when it's implicitly not always visible
+			if inventory and inventory.is_visible():
+				inventory.close()
```

Comment:
I'm not too happy about the implementation of the inventory being so closely tied to the action menu. What if a (to be implemented) two-click interface is being used, instead of a verb or action menu, and we want the inventory to be closed when the menu is opened or if an action menu is used and the developer doesn't want an auto-closeable inventory? Would it be better to have an exported `auto_closeable` variable in `inventory` instead?

I don't remember if we already make similar assumptions elsewhere. If so, this has to be revisited anyway, so I'm fine with it.

Reply:
I don't really like this hack either, though https://github.com/godotengine/escoria/blob/master/device/globals/game.gd#L33 is a similar thing.

But I think I see where you're going with this now that I looked at the code.

Because `load_hud()` looks at the settings for an inventory already, and injects it into the scene tree if it's not there (like it would be in old SCUMM and Daiza), the default for `auto_closeable` would be `false` for Daiza and the demo scene but I'd set it to `true`.

I might opt for a more verbose name like `hidden_by_default`.

Did I understand correctly?

Reply:
> Because `load_hud()` looks at the settings for an inventory already, and injects it into the scene tree if it's not there (like it would be in old SCUMM and Daiza), the default for `auto_closeable` would be `false` for Daiza and the demo scene but I'd set it to `true`.

It does, but I removed that code in #114. Since there's already code like this present, I think it's fine for this PR though, and I can change that in my PR instead. Yes, I agree that `hidden_by_default` is a better name.

Reply:
Nice, thanks!

Reply:
But forgot to ask, do you want the name and stuff in this PR still?

Reply:
No, just leave it like this, and I'll merge it once I've tested it.